src/WikiService.Data/WikiService.Data.csproj

```diff
@@ -8,4 +8,8 @@
 		<ProjectReference Include="..\WikiService.Models.Dto\WikiService.Models.Dto.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <PackageReference Update="LT.DigitalOffice.Kernel" Version="1.1.61" />
```

Comment:
in targets

Reply:
When I use version 1.1.63, I get error in stranger code (in EditArticleCommand).